stencil-library/src/components/subaccount-details/shared-subaccount-details.scss

```diff
@@ -0,0 +1,7 @@
+@import 'reboot';
+@import 'bootstrap-utilities';
+@import 'grid';
+
+.flex-1 {
```

Comment:
I don't believe this is being used. 

You may also want to include the following, which just makes sure that some fonts and other things are set consistently across components:
```
:host {
  @include host-base-styles;
}
```

https://github.com/search?q=repo%3Ajustifi-tech%2Fweb-component-library%20host-base-styles&type=code